public/app/features/query/state/DashboardQueryRunner/UnifiedAlertStatesWorker.test.ts

```diff
@@ -88,89 +92,79 @@ describe('UnifiedAlertStatesWorker', () => {
 
   describe('when run is called with incorrect props', () => {
     it('then it should return the correct results', async () => {
-      const { getMock, options } = getTestContext();
+      const { options } = getTestContext();
       const dashboard = createDashboardModelFixture({});
 
       await expect(worker.work({ ...options, dashboard })).toEmitValuesWith((received) => {
         expect(received).toHaveLength(1);
         const results = received[0];
         expect(results).toEqual({ alertStates: [], annotations: [] });
-        expect(getMock).not.toHaveBeenCalled();
       });
     });
   });
 
   describe('when run repeatedly for the same dashboard and no alert rules are found', () => {
+    const nameSpaces = [mockPromRuleNamespace({ groups: [] })];
+    const { dispatchMock, options } = getTestContext();
+    //getMock.mockResolvedValue(getResults);
```

Comment:
Should we remove this or is it work-in-progress?

Reply:
yes